examples/bpk-component-page-indicator/examples.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+
+import BpkPageIndicator from '../../packages/bpk-component-page-indicator';
+import { BpkBackgroundImage } from '../../packages/bpk-component-image';
+import { cssModules } from '../../packages/bpk-react-utils';
+
+import STYLES from './examples.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+const image =
+  'https://content.skyscnr.com/96508dbac15a2895b0147dc7e7f9ad30/canadian-rockies-canada.jpg';
+const imageWidth = 612;
+const imageHeight = 408;
+const PageIndicatorContainer = (props) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  return (
+    <BpkPageIndicator
+      currentIndex={currentIndex}
+      onClick={(_e, index) => {
+        setCurrentIndex(index);
+      }}
+      bulletLabel="Go to slide"
+      prevNavLabel="Previous slide"
+      nextNavLabel="Next slide"
+      {...props}
+    />
+  );
+};
+
+const DefaultExample = () => (
+  <PageIndicatorContainer id="default-example" totalBullets={7} />
+);
+
+const ThreePagesExample = () => (
+  <PageIndicatorContainer id="three-pages-example" totalBullets={3} />
+);
+
+const OverImageExample = () => (
+  <BpkBackgroundImage
+    width={612}
+    height={408}
+    style={{ width: imageWidth, height: imageHeight }}
+    imageStyle={{
+      backgroundRepeat: 'no-repeat',
+      backgroundSize: 'cover',
+      backgroundPosition: '50% 50%',
+    }}
+    src={image}
+  >
+    <PageIndicatorContainer
+      id="over-image-example"
```

Comment:
`id` also needs to be removed

Reply:
Have removed `id`, thank you for your patient review!